src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizard.cmp

```diff
@@ -0,0 +1,254 @@
+                    <aura:if isTrue="{!not(v.isReadOnly)}">
+                        <div class="slds-text-align_center slds-m-vertical_medium">
+                            <h2 class="slds-text-heading_medium">{!v.templateMetadata.pageHeader}</h2>
+                        </div>
+                        <aura:set attribute="else">
+                            <div class="slds-section slds-is-open slds-p-bottom_small">
+                                <h3 class="slds-section__title slds-theme_shade">
+                                    <span class="slds-truncate slds-p-horizontal_small"
+                                          title="{!$Label.c.bgeBatchTemplateOverview}">
+                                            {!$Label.c.bgeBatchTemplateOverview}
+                                    </span>
+                                </h3>
+                            </div>
+                        </aura:set>
+                    </aura:if>
+
+                    <!--Progress Indicator-->
+                    <aura:if isTrue="{!not(v.isReadOnly)}">
+                        <div class="slds-p-bottom_large">
+                            <lightning:progressIndicator currentStep="{!v.templateMetadata.progressIndicatorStep}" type="base"
+                                                         hasError="false" variant="base">
+                                <lightning:progressStep label="{!v.templateMetadata.labels.recordInfoLabel}" value="1"/>
+                                <lightning:progressStep label="{!$Label.c.bgeBatchTemplateSelectFields}" value="3"/>
+                                <lightning:progressStep label="{!$Label.c.bgeBatchTemplateSetFieldOptions}" value="4"/>
+                            </lightning:progressIndicator>
+                        </div>
+                    </aura:if>
+                </aura:if>
+
+                <!--Template Container-->
+
+                <!-- Step 1: Overview -->
+                <aura:if isTrue="{!v.templateMetadata.progressIndicatorStep == '1' }">
+                    <lightning:layout horizontalAlign="center">
+                        <lightning:layoutItem size="10">
+                            <lightning:input type="text" name="name" label="{!$Label.c.stgLabelName}"
+                                             value="{!v.templateInfo.name}" readonly="{!v.isReadOnly}" required="true"/>
+                            <lightning:textarea name="description" label="{!v.templateMetadata.labels.description}"
+                                                value="{!v.templateInfo.description}" readonly="{!v.isReadOnly}"
+                                                required="true" class="slds-p-top_medium"/>
+                            <div class="slds-p-top_large">
+                                <aura:if isTrue="{!v.isReadOnly}">
+                                    <ui:outputCheckbox value="{!v.templateInfo.requireTotalMatch}" class="slds-form-element__control"/>
```

Comment:
Add inputs for Total Expected Gifts and Total Expected Amount above the checkbox

Reply:
Oh wait, this belongs with the read/write version on line 144.

Reply:
Done!